zulip_bots/zulip_bots/bots/mention/mention.py

```diff
@@ -2,13 +2,31 @@
 
 import requests
 from typing import Any, List
+import logging
 
 class MentionHandler(object):
     def initialize(self, bot_handler: Any) -> None:
         self.config_info = bot_handler.get_config_info('mention')
         self.access_token = self.config_info['access_token']
         self.account_id = ''
 
+        self.check_access_token()
+
+    def check_access_token(self) -> None:
+        test_query_header = {
+            'Authorization': 'Bearer ' + self.access_token,
+            'Accept-Version': '1.15',
+        }
+        test_query_response = requests.get('https://api.mention.net/api/accounts/me', headers=test_query_header)
+
+        try:
+            test_query_data = test_query_response.json()
+            if test_query_data['error'] == 'invalid_grant' and \
+               test_query_data['error_description'] == 'The access token provided is invalid.':
```

Comment:
Are there cases where the error is "invalid_grant", but the error description is not "the access token provided is invalid"?

Reply:
I really didn't find any other errors in my testing or in the documentation with the error of 'invalid_grant'. I did this just to be sure that the reason for the error is the incorrect API Key.